OpenMarket/OpenMarket/View/CollectionView/Cell/ProductGridCell.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  ProductGridCell.swift
+//  OpenMarket
+//
+//  Created by Ayaan, junho on 2022/11/22.
+//
+
+import UIKit
+
+final class ProductGridCell: UICollectionViewCell {
+    //MARK: - Views
+    private let thumbnailImageView: UIImageView = {
+        let imageView: UIImageView = UIImageView()
+        
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    private let nameLabel: UILabel = {
+        let label: UILabel = UILabel()
+        
+        label.font = UIFont.preferredFont(forTextStyle: .title3,
+                                          compatibleWith: UITraitCollection.init(preferredContentSizeCategory: .large))
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }()
+    private let stockLabel: StockLabel = StockLabel()
+    private let priceLabel: PriceLabel = PriceLabel()
+    private let stackView: UIStackView = {
+        let stackView: UIStackView = UIStackView()
+        
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .fill
+        stackView.spacing = 10
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return stackView
+    }()
+    
+    private var product: Product? {
+        didSet {
+            setupDataIfNeeded()
+        }
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    //MARK: - Method
+    func updateWithProduct(_ newProduct: Product) {
+        self.product = newProduct
+    }
+    
+    private func configure() {
+        setUpLayer()
+        setUpTextAlignment()
+        setupViewsIfNeeded()
```

Comment:
```suggestion
        setUpTextAlignment()
        setupViewsIfNeeded()
```
위 아래 setUp과 setup을 다른 컨벤션으로 가져가신건 의도 하신걸까요?

Reply:
아뇨 의도는 없었습니다. 일관성 있게 네이밍하지 못했네요. 전체적으로 확인해서 `setUp` 으로 모두 고쳤습니다.
[43682a9](https://github.com/yagom-academy/ios-open-market/pull/218/commits/43682a91bf6275efd3c7f7daebb970d1a65fa61c)